app/src/main/java/com/github/polybooks/core/database/implementation/OLBookDatabase.kt

```diff
@@ -35,6 +35,7 @@ private const val PUBLISH_DATE_FIELD_NAME = "publish_date"
 private const val AUTHOR_NAME_FIELD_NAME = "name"
 
 private const val DATE_FORMAT = "MMM dd, yyyy"
+private const val DATE_FORMAT2 = "yyyy"
```

Comment:
How does this correct the time zone problem? It stores just year right?

Reply:
This is not about the timezone issue. This issue is about OpenLibrary not always having a precise publish date for a book

Reply:
To answer more precisely your question: I need two date formats because OpenLibrary sometimes gives me a date with the first format, and sometime juste a year with the second format. And I need to be able to parse the two into a Date.

Reply:
ok it makes sense!